lib/screens/home/tabs/profile/profile.dart

```diff
@@ -38,29 +41,84 @@ class _ProfileScreenState extends State<ProfileScreen> {
-        mainAxisSize: MainAxisSize.min,
-        children: <Widget>[
-          Icon(Icons.check_circle_outline),
-          Text(
-            user.email,
-            textAlign: TextAlign.center,
-          ),
-        ],
-      ),
-    );
+  Widget _buildSuccess(BuildContext context) {
+    double width = MediaQuery.of(context).size.width;
+    double height = MediaQuery.of(context).size.height;
+    double heightFactor = height / 1000;
+    return DefaultTextStyle(
+        style: GoogleFonts.roboto().copyWith(color: C.primaryUnHighlightedColor),
+        child: Stack(children: [
+          ScreenBackground(elementId: 0),
+          Container(
+              child: Column(mainAxisSize: MainAxisSize.min, children: [
+            SizedBox(height: height / 6),
+            Flexible(
+                flex: 4,
+                child: Column(children: [
+                  Container(
+                      child: Image.asset(
+                    S.assetEcellLogoWhite,
+                    width: width * 0.35 * heightFactor,
+                  )),
+                  SizedBox(height: height / 20),
+                  Form(
+                      child: Padding(
+                          padding: EdgeInsets.only(
+                              left: D.horizontalPadding, right: D.horizontalPadding),
+                          child: Column(
+                              crossAxisAlignment: CrossAxisAlignment.start,
+                              children: <Widget>[
+                                _label("First Name", heightFactor),
+                                _labelValue(width, height, user.firstName),
+                                SizedBox(height: height / 30),
+                                _label("Last Name", heightFactor),
+                                _labelValue(width, height, user.lastName),
+                                SizedBox(height: height / 30),
+                                _label("Email Address", heightFactor),
+                                _labelValue(width, height, user.email),
+                                SizedBox(height: height / 30),
+                                _label("Phone Number", heightFactor),
+                                _labelValue(width, height, user.phoneNumber),
+                                SizedBox(height: height / 15),
+                                Container(
+                                    alignment: Alignment.topRight,
+                                    child: GestureDetector(
+                                        onTap: () {}, //TODO
+                                        child: Text("Change Number?",
+                                            style: TextStyle(
+                                              color: C.primaryHighlightedColor,
+                                              fontSize: 20 * heightFactor,
+                                            ))))
+                              ])))
+                ]))
+          ]))
+        ]));
```

Comment:
commas after each closing bracket.

Reply:
i did that as you said it reduces a line. will add now onwards